doc-content/enterprise-only/employee-rostering/er-rotation-con.adoc

```diff
@@ -5,15 +5,13 @@ The employee rostering application supports a _rotation pattern_ for shifts and
 
 The rotation pattern is a "model" period of any time starting from two days. The pattern is not tied to a particular date. 
 
-You can set the shift times for every day of the rotation. Optionally, the names of the default employees for each shift in each day can also be set.
-
-NOTE: In the current version, you cannot enter the names of employees in the rotation period. However, you can see the operation of rotation periods with default employees in the example tenants.
+You can create _shift templates_ for every day of the rotation. Every shift template sets the time of a shift. Optionally, the template can include the name of the default employee for the shift.
```

Comment:
What is edited is a single object called "the rotation". It can become "a rotation" when it is applied multiple times, but while it's being edited there is only one rotation.

Besides, "day of rotation" sounds like "day something rotates".